Update packages for aur

aboe76 opened the pull request:
facter 1.6.17
puppet 3.0.2
hiera 1.1.2
puppetdb-git added

GregSutcliffe committed:
Merge pull request #4 from aboe76/facter_1.6.14

updated to facter 1.6.14

aboe76 committed:
Merge remote-tracking branch 'upstream/master'

aboe76 committed:
Merge remote-tracking branch 'upstream/master'

aboe76 committed:
Merge remote-tracking branch 'upstream/master'

aboe76 committed:
Merge remote-tracking branch 'upstream/master'

aboe76 committed:
Updated facter to 1.6.17
Updated puppet to 3.0.2
Updated ruby-hiera to 1.1.2
added puppetdb-git

and removed .kdev4

aboe76 committed:
updated rakefile

aboe76 committed:
updated Rakefile

GregSutcliffe commented:
Sorry for the delay. Christmas was crazy. Going to merge this now - but I'm not sure I want to be maintainer for puppetdb-git. I don't use puppetdb at all, so it's impossible for me to test it. Do you want to submit that separately?

GregSutcliffe commented:
Merged without puppetdb-git, since you already own that package :). Thanks!

aboe76 commented:
I wanted to stash my puppetdb-git package somewhere seemed like a good
idea, you don't have to maintain puppetdb on aur, I will take care of it.
maybe just leave it in the repo of puppet packages, they seem to belong to
each other...hehehe.

2013/1/11 Greg Sutcliffe notifications@github.com

> Merged without puppetdb-git, since you already own that package :). Thanks!
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/GregSutcliffe/aur-projects/pull/6#issuecomment-12160062.

aboe76 committed:
updated depencies:
ruby-json_pure this is needed to fix
Hiera terminus not supported without hiera library
